app/models/manageiq/providers/ansible_runner_workflow.rb

```diff
@@ -181,6 +181,14 @@ def checkout_git_repository
     save!
     _log.info("Checking out git repository to #{options[:git_checkout_tempdir].inspect}...")
     css.checkout_git_repository(options[:git_checkout_tempdir])
+  rescue MiqException::MiqUnreachableError => err
+    miq_task.job.timeout!
+    r_id = options[:extra_vars]["manageiq"]["request"].split('/').last.to_i
```

Comment:
Not sure if there is a better way to find this ID.  Don't know of a better idea myself so don't change unless you do, but making not of it since it is a bit weird.

Reply:
Hi @NickLaMuro Thanks for working with Drew on this.  I agree that we should add retry logic in a separate PR.
I don't know of a better way to get the request ID.

Reply:
Will `options[:extra_vars]` always exist with those keys and yield an array? Maybe it could be 
`options.dig(:extra_vars, "manageiq", "request").try(:split, '/').try(:last).to_i`